### PR TITLE
Csv export

### PR DESCRIPTION
**Pull request by mayerm94**
Add make task 'make csv_export` which will output the contents of the `endpoint_export` database view into a local csv.

Pull requests into this repository require the following checks to be completed by the submitter and reviewers.

**Submitter:**
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR.
  - The JIRA ticket for this PR is:  https://oncprojectracking.healthit.gov/support/browse/LANTERN-166
- [x] Tests are complete. NA
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.
- [x] ~`make update_mods branch=<your_PR_branch>` has been run.~
  - ~The resulting go.mod and go.sum files are the only files in the final commit before the PR is merged~ NA

**Primary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

**Secondary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code accomplishes the tasks purpose.
- [x] Tests are complete.
  - Tests have been run locally and pass.

**Merger:**

Prior to merging, ensure that all checks have been completed. Delete the branch after the merge is complete.

### CODE REVIEW

File: endpointmanager/cmd/endpointExporter/main.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package main
```
Comment (edeyoung) [inline endpointmanager/cmd/endpointExporter/main.go:1]: stupidly minor consistency thing: don't capitalize 'Exporter' in filename.
